dashboard/test/controllers/teacher_dashboard_controller_test.rb

```diff
@@ -11,20 +11,23 @@ class TeacherDashboardControllerTest < ActionController::TestCase
 
   test 'index: returns forbidden if no logged in user' do
     get :show, params: {section_id: @section.id}
-    assert_redirected_to_sign_in
+    assert_response :redirect
+    assert_redirected_to "http://test.host/home"
   end
 
   test 'index: returns forbidden if logged in user is not a teacher' do
     sign_in create(:student)
     get :show, params: {section_id: @section.id}
-    assert_response :forbidden
+    assert_response :redirect
+    assert_redirected_to "http://test.host/home"
   end
 
   test 'index: returns forbidden if requested section does not belong to teacher' do
     sign_in @section_owner
     other_teacher_section = create :section
     get :show, params: {section_id: other_teacher_section.id}
-    assert_response :forbidden
+    assert_response :redirect
+    assert_redirected_to "http://test.host/home"
```

Comment:
Can we also add a test that it goes to the correct course page as well as these home redirects?